code/modules/mob/new_player/new_player.dm

```diff
@@ -501,7 +501,7 @@
 /mob/new_player/hear_say(var/message, var/verb = "says", var/datum/language/language = null, var/alt_name = "",var/italics = 0, var/mob/speaker = null)
 	return
 
-/mob/new_player/hear_radio(var/message, var/verb="says", var/datum/language/language=null, var/part_a, var/part_b, var/mob/speaker = null, var/hard_to_hear = 0)
+/mob/new_player/hear_radio()

```

Comment:
``hear_radio(thing, differentThing, anotherThing, oneMoreThing)``
is fine
``hear_radio(thing = someThing)``
is not

The code currently only uses calls without named arguments. The first form. So yes, it won't runtime.